### PR TITLE
Add CitycraftLauncher 1.9.9

### PR DESCRIPTION
**Pull request by simon511000**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1401)

### CODE REVIEW

File: manifests/Simon511000/Citycraftlauncher/1.9.9.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Id: simon511000.citycraftlauncher
```
Comment (MarNicGit) [inline manifests/Simon511000/Citycraftlauncher/1.9.9.yaml:1]:
```suggestion
Id: Simon511000.CityCraftLauncher
```

We generally use Pascal casing for the packages.

**Comment by simon511000** [comment]
Microsoft removed the Microsoft Defender SmartScreen error.

> We have removed the detection.  Please follow the steps below to clear cached detection and obtain the latest malware definitions.
> 
>      1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender 
>      2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
>      3. Run "MpCmdRun.exe -SignatureUpdate"
> 
> Alternatively, the latest definition is available for download here: https://www.microsoft.com/en-us/wdsi/definitions
> 
> Thank you for contacting Microsoft.

**Comment by simon511000** [comment]
image

**Comment by simon511000** [comment]
I confirm that there is no more SmartScreen error!

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**